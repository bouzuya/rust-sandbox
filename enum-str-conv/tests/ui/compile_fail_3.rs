fn main() {
    #[derive(enum_str_conv::EnumStrConv)]
    #[enum_str_conv]
    enum MyEnum {
        #[enum_str_conv(str = "apple")]
        A,
    }
}
