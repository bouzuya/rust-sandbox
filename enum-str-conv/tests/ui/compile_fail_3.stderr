error: expected attribute arguments: #[enum_str_conv(error = ..., unknown = ...)]
 --> tests/ui/compile_fail_3.rs:3:7
  |
3 |     #[enum_str_conv]
  |       ^^^^^^^^^^^^^
